server/endpoint/endpoint.go

```diff
@@ -0,0 +1,73 @@
+package endpoint
+
+import (
+	"github.com/giantswarm/microendpoint/endpoint/healthz"
+	"github.com/giantswarm/microendpoint/endpoint/version"
+	healthzservice "github.com/giantswarm/microendpoint/service/healthz"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/node-operator/server/middleware"
+	"github.com/giantswarm/node-operator/service"
+)
+
+// Config represents the configuration used to create a endpoint.
+type Config struct {
+	// Dependencies.
```

Comment:
Maybe drop the comment as agreed earlier?

Reply:
Done.